indexed_warning_files.json

```diff
@@ -0,0 +1 @@
a.cc", "479": "tensorflow/python/eager/pywrap_tfe_src.cc", "480": "tensorflow/compiler/mlir/lite/utils/lstm_utils.cc", "481": "tensorflow/compiler/mlir/lite/transforms/prepare_composite_functions_tf.cc", "482": "tensorflow/compiler/mlir/lite/transforms/dilated_conv.cc", "483": "tensorflow/lite/delegates/nnapi/quant_lstm_sup.cc", "484": "tensorflow/compiler/mlir/lite/transforms/dense_to_sparse.cc", "485": "tensorflow/compiler/mlir/lite/transforms/lower_static_tensor_list.cc", "486": "tensorflow/lite/delegates/nnapi/nnapi_delegate.cc", "487": "tensorflow/compiler/mlir/lite/transforms/prepare_tf.cc", "488": "tensorflow/compiler/mlir/lite/transforms/legalize_tf.cc", "489": "tensorflow/compiler/mlir/lite/transforms/optimize.cc", "490": "tensorflow/lite/python/interpreter_wrapper/interpreter_wrapper.cc", "491": "tensorflow/lite/tools/verifier.cc", "492": "tensorflow/lite/tools/optimize/model_utils.cc", "493": "tensorflow/lite/tools/optimize/quantization_utils.cc", "494": "tensorflow/lite/tools/versioning/op_version.cc", "495": "tensorflow/lite/tools/versioning/runtime_version.cc", "496": "tensorflow/lite/tools/optimize/quantize_model.cc", "497": "tensorflow/lite/tools/optimize/quantize_weights.cc", "498": "tensorflow/lite/python/optimize/calibration_wrapper.cc", "499": "tensorflow/lite/toco/tflite/import.cc", "500": "tensorflow/compiler/mlir/lite/flatbuffer_import.cc", "501": "tensorflow/compiler/mlir/lite/ir/tfl_ops.cc", "502": "tensorflow/compiler/mlir/lite/flatbuffer_export.cc", "503": "tensorflow/compiler/mlir/lite/python/saved_model_to_tfl_flatbuffer.cc", "504": "tensorflow/lite/experimental/microfrontend/lib/fft.cc", "505": "tensorflow/core/tpu/tpu_embedding_optimization_parameters_utils.cc", "506": "tensorflow/core/platform/default/logging.cc", "507": "tensorflow/core/platform/default/logging.cc", "508": "tensorflow/core/platform/protobuf.cc", "509": "tensorflow/core/framework/allocator_registry.cc", "510": "tensorflow/core/framework/cpu_allocator_impl.cc", "511": "tensorflow/core/kernels/lookup_util.cc", "512": "tensorflow/lite/delegates/nnapi/nnapi_delegate.cc", "513": "tensorflow/core/framework/allocator_registry.cc", "514": "tensorflow/core/common_runtime/lower_case_op.cc", "515": "tensorflow/core/data/service/compression_utils.cc", "516": "tensorflow/core/kernels/data/prefetch_autotuner.cc", "517": "tensorflow/lite/tools/versioning/runtime_version.cc", "518": "tensorflow/compiler/xla/service/hlo_evaluator.cc", "519": "tensorflow/compiler/tf2xla/xla_resource.cc", "520": "tensorflow/core/kernels/stack.cc", "521": "tensorflow/core/profiler/internal/parse_annotation.cc", "522": "tensorflow/core/framework/cpu_allocator_impl.cc", "523": "tensorflow/lite/experimental/microfrontend/lib/fft.cc", "524": "tensorflow/lite/toco/graph_transformations/ensure_uint8_weights_safe_for_fast_int8_kernels.cc", "525": "tensorflow/compiler/xla/service/while_loop_invariant_code_motion.cc", "526": "tensorflow/core/tpu/tpu_embedding_optimization_parameters_utils.cc"}
```

Comment:
You might want to add this file to `.git/config/ignore` or use `git add ..` instead of `git commit -a ...`